dependencies.yaml

```diff
@@ -391,6 +394,12 @@ dependencies:
         packages:
           - pyproj>=3.6.0,<3.7a0
 
+  depends_on_libcuspatial:
+    common:
+      - output_types: conda
+        packages:
+          - &libcuspatial_unsuffixed libcuspatial==24.10.*,>=0.0.0a0
```

Comment:
#1450 will add requirements / pyproject sections to this. They don't need to exist until `libcuspatial` wheels exist.